### PR TITLE
Update Wordpress template

### PR DESCRIPTION
**Pull request by janeczku**
- Fixes issue running on smaller VMs due to DB using a lot of memory (was ~ 256MB, now  ~ 128 MB).
- Use `retain_IP` for the db
- Allow to configure `innodb_buffer_pool_size`

### CODE REVIEW

**Comment by rawmind0** [comment]
Hi @janeczku  .. thanks for your PR ..

Have you consider to use the galera cluster package instead add the database inseide the package?? You could deploy it and link your service with it, and take profit of the already created packaged. What do you think??

Best regards....

File: templates/wordpress/1/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      label: "MySQL root password"
+      required: true
+      default: "wordpress"
+      type: "string"
+    - variable: "innodb_buffer_pool_size"
+      description: "Optionally tweak the InnoDB buffer pool size configuration setting"
+      label: "MySQL InnoDB Buffer Pool Size"
+      required: true
+      default: "8M"
+      type: "string"
```
Comment (rawmind0) [inline templates/wordpress/1/rancher-compose.yml]: # Add service combo to select database service. You need to deploy it before.
    - variable: "wordpress_db_service"
      description: |
        Choose the database service to use. 
        It's really recommended a dedicated database service just for wordpress.
      label: "Database stack/service"
      default: "galera/galera-lb"
      required: true
      type: "service"

File: templates/wordpress/1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+wordpress:
+  image: wordpress
+  links:
+    - db:mysql
```
Comment (rawmind0) [inline templates/wordpress/1/docker-compose.yml]: # Set the link to database service
-- - db:mysql
++ - ${wordpress_db_service}:mysql

File: templates/wordpress/1/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  ports:
+    - ${public_port}:80
+
```
Comment (rawmind0) [inline templates/wordpress/1/docker-compose.yml]: # Remove database definition. Re-use the galera cluster catalog package.

File: templates/wordpress/1/rancher-compose.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+db:
```
Comment (rawmind0) [inline templates/wordpress/1/rancher-compose.yml]: # Remove db scale definition

**Review by rawmind0 — CHANGES_REQUESTED** [review]

**Comment by cjellick** [comment]
Very stale. Closing. Thanks! Please open a new PR if there is still a relevant change.

**Pull request closed without merging**